components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -30,242 +27,273 @@
       </ul>
     ]]></string>
 
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
+    <string name="error_malformed_uri_title_alternative">The address isn’t valid</string>
+    <!-- This string contains markup. The URL should not be localized. -->
+    <string name="error_malformed_uri_message_alternative"><![CDATA[
+      <ul>
+        <li>Web addresses are usually written like <strong>http://www.example.com/</strong></li>
+        <li>Make sure that you’re using forward slashes (i.e. <strong>/</strong>).</li>
+      </ul>
+    ]]></string>
 
-    <string name="error_malformedURI_title">The address isn’t valid</string>
-    <string name="error_malformedURI_message"><![CDATA[
-<ul>
-  <li>Web addresses are usually written like <strong>http://www.example.com/</strong></li>
-  <li>Make sure that you’re using forward slashes (i.e. <strong>/</strong>).</li>
-</ul>
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
+    <string name="error_redirect_loop_title_alternative">The page isn’t redirecting properly</string>
+    <string name="error_redirect_loop_message_alternative"><![CDATA[
+      <ul>
+        <li>This problem can sometimes be caused by disabling or refusing to accept cookies.</li>
+      </ul>
     ]]></string>
 
-    <string name="error_redirectLoop_title">The page isn’t redirecting properly</string>
-    <string name="error_redirectLoop_message"><![CDATA[
-<ul>
-  <li>This problem can sometimes be caused by disabling or refusing to accept cookies.</li>
-</ul>
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
+    <string name="error_unsupported_protocol_title">The address wasn’t understood</string>
```

Comment:
it looks like `mozac_browser_errorpages_unknown_protocol_title` and `mozac_browser_errorpages_unknown_protocol_message` are [the ones used now](https://github.com/mozilla-mobile/android-components/blob/af881f8/components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt#L114-L115) (I'll make this change in a PR I'm about to open)